polynote-server/src/main/scala/polynote/server/Server.scala

```diff
@@ -150,8 +150,8 @@ class Server {
 
         gzipped orElse nogzip orElse fromJar
       }.catchAll {
-        case err: HTTPError => ZIO.fail(err)
-        case err => Logging.error("Error serving file", err) *> ZIO.fail(InternalServerError("Error serving file", Some(err)))
+        case err: HTTPError => Logging.error(s"Error serving file: $name", err) *> ZIO.fail(err)
+        case err => Logging.error(s"Error serving file: $name", err) *> ZIO.fail(InternalServerError(s"Error serving file: $name", Some(err)))
```

Comment:
This is an unrelated change but helped me diagnose some issues I ran into after not running Polynote locally for a while 😅